libcontainer/integration/utils_test.go

```diff
@@ -19,6 +20,36 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var busyboxTar string
+
+// init makes sure the container images are downloaded,
+// and initializes busyboxTar. If images can't be downloaded,
+// we are unable to run any tests, so panic.
+func init() {
+	// Figure out path to get-images.sh. Note it won't work
+	// in case the compiled test binary is moved elsewhere.
+	_, ex, _, _ := runtime.Caller(0)
+	getImages, err := filepath.Abs(filepath.Join(filepath.Dir(ex), "..", "..", "tests", "integration", "get-images.sh"))
+	if err != nil {
+		panic(err)
```

Comment:
Shouldn't this cause t.Skip rather than panic ?

Reply:
You can't cause `t.Skip` because it's done in `init()`.

Reply:
We can, by defining a global variable

Reply:
You would want it to be `t.Fatal` because the images are needed for tests (and skipping them would hide test failures), and `t.Fatal` will kill the test anyway. I guess it is slightly nicer to do that than `panic`, but this really shouldn't happen often enough to be an issue (not to mention you'd need to add `t.Fatal` to some helper function and I can't see an obvious candidate in `utils_test.go` -- none of the functions take `*testing.T`).